<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../background pics/icons/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-11</x>
      <y>-1</y>
      <width>1031</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 120, 0);
color:white</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../background pics/propertyPics/DEVOE partners logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="Loginwidget" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>150</y>
       <width>351</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 120, 0,0.7);
color: white;</string>
     </property>
     <widget class="QTextEdit" name="txtUserName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#txtUserName{
background-color:rgb(255, 255, 255);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="pswd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto Medium</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: none;</string>
      </property>
      <property name="text">
       <string>UserName:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginbtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#loginbtn{
background-color:rgb(171, 255, 14);
color: rgb(0, 120, 0);
font: 16pt &quot;Roboto&quot;;
filter: box-shadow(100px 100px red);
}</string>
      </property>
      <property name="text">
       <string>SignUp</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtPassword">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#txtPassword{
background-color:rgb(255, 255, 255);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="pswd_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto Medium</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: none;</string>
      </property>
      <property name="text">
       <string>Passward:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtEmail">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#txtEmail{
background-color:rgb(255, 255, 255);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="pswd_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto Medium</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: none;</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Roboto&quot;;
color: rgb(255, 255, 255);
background-color:none;</string>
     </property>
     <property name="text">
      <string>Har Pata, Hamein Pata Hai</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>800</width>
       <height>500</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../propertyPics/construction-concept-with-engineering-tools.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>label_3</zorder>
    <zorder>Loginwidget</zorder>
    <zorder>label_4</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
